features/comments_and_kudos/image_safety_mode.feature

```diff
@@ -0,0 +1,62 @@
+Feature: Image safety mode
+  In order to protect users
+  As a site owner
+  I'd like to control which comments can include images
+
+  Scenario Outline: Images are embedded in comments when image safety mode is off.
+    Given the setup for testing image safety mode on <commentable>
+      And image safety mode is disabled for comments
+    When I view <commentable> with comments
+    Then I should see the image "src" text "https://example.com/image.jpg"
+      And I should not see "OMG! https://example.com/image.jpg"
```

Comment:
The test added to add_comment.feature in #4729 can be removed, because this tests the same.